### ISSUE TITLE
Stratification

### ISSUE THREAD

**Issue by kurtisanstey** [kurtisanstey/project#22]
In attempting a depth analysis it is important to consider the effects of stratification on the observed spectra. 

**Currently**

- Using CTD data from Line P Station P4 (~26 km WNW from Barkley Canyon).
- Assuming fairly uniform stratification below -200 m. 
- Sample summer plot: https://github.com/kurtisanstey/project/blob/master/plots/N2_plots/N2_lower_Summer_2018.pdf
- Sample winter plot: https://github.com/kurtisanstey/project/blob/master/plots/N2_plots/N2_lower_Winter_2018.pdf
- Between -200 m and -900 m, for 2009 - 2019, finding a depth average N^2 of 1.11e-5 (rad/s)^2. 

**To do**

- [x] Get a DFO Water Properties account. Access has been granted.
- [x] Get closest La Perouse CTD data, rather than Line P. Retrieved data for 2013, 2014, 2017, and 2018 at station LB14, the closest to Barkley Canyon with sufficient depth (-1180 m) at ~21 km to the SE. This is closer than the previous Line P station, and is situated in a similar position on the slope as the section of Barkley Canyon being investigated.
- [x] Smooth N^2 data and average a few profiles, to get a better picture of depth dependence.
- [x] Re-evaluate stratification and consider WKB scaling to mitigate its effects, if necessary (ask Jody about this).

**Comment by jklymak** [comment]
Hi Kurtis. I don't think the stratification is as "nearly constant" as you think.  It definitely decays with depth.  The spiky thing you are plotting has substantial contamination from the internal waves.   Suggest smoothing over the spikes and averaging a few profiles.

**Comment by kurtisanstey** [comment]
@jklymak  

- Retrieved CTD data from two La Perouse cruises per year, in each of 2013, 2014, 2017, and 2018, at station LB14, ~21 km to the SE of Barkley Canyon, to a depth of -1180 m.
- Closer than previously used Line P station P4 (which was ~26 km away), and is situated in a similar position on the slope as the segment of Barkley Canyon between Axis and Upper Slope. 
- Calculated and smoothed N^2 data, and averaged the obtained profiles.
- There is significant stratification above -200 m, peaking around 5.5e-4 (rad/s)^2 at top of the pycnocline.
- Between -200 and -900 m, N^2 degrades from about 1.75e-5 to 0.75e-5 (rad/s)^2. 
- **Question**: Is this enough variation to warrant scaling the data? My first thought would be to scale between -100 and -200 m, at least - the upper limit of my ADCP data. Below that the gradient seems negligible, by comparison.

![image](https://user-images.githubusercontent.com/57200139/110750335-2b66e180-81f7-11eb-92ee-cb25187cf93b.png)

**Comment by jklymak** [comment]
- don't try and use that version - it is still too wavy.  Looks like you just took one cast?  
- Yes, N^2 varies by a factor of 2.3.  That means your GM spectrum will vary by the same factor...
- This is a super-standard thing to do _ I would guess most of the plots you have seen in papers do this.
- Note you only scale by one depth-dependent factor - you dont' try to get a different N^2 for every time.  And all you do is multiply the factor times the spectrum.  For 2-D spectra its a bit harder because z needs to be scaled as well.

**Comment by kurtisanstey** [comment]
Great advice, thank you! No need to respond until our meeting; I'm TAing until then.

- I smoothed and then averaged eight casts, two from each year (2013, 2014, 2017, 2018). 
- I added the depth-dependent scaling profile (right-most plot) based on the smoothed N. This assumes using a single average of the lower depth N (between -894 and -904 m) to get a 'base' N for the GM profile. Next I'll apply the scaling to my spectra so they're comparable to the common N value. 

![image](https://user-images.githubusercontent.com/57200139/111415400-12e74300-869f-11eb-8cd4-582c4bb02dc3.png)
